### ISSUE TITLE
Main is too strict on location

### ISSUE THREAD

**Issue by ChristopherKiss** [bennymeg/nx-electron#46]
**Describe the bug**
I've changed my `dist` folder to `build`, and running package fails since the output path is hard coded.

**To Reproduce**
* Change references from `dist` to `build`.
* Package the app.
* Try launching the exe that was packages.

**Expected behavior**
I should be able to package the app and run it.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows 10, 64 bit
 - Nx Electron Version: 9.3.2

**Additional context**
The output is hard coded:

https://github.com/bennymeg/nx-electron/blob/863812706dcd338571dab5ec1fcd324271fd801b/src/builders/package/package.impl.ts#L58

My output path would look like: /build/apps/{appName}/main

There is no way to change dist to build, nor could I rename main either if I wanted to. This path should be configurable for people that intend to change the output path or even the main file.

It should piece this together from angular.json outputPath and main properties.

Simplified copy of angular.json

```
 "app-name": 
            "architect":
                "build":
                    "builder": "nx-electron:build",
                    "options":
                        "outputPath": "build/apps/app-name",
                        "main": "apps/app-name/src/main.ts",
```

**Comment by bennymeg** [comment]
Sounds good, Ill look into it.

**Comment by bennymeg** [comment]
I've looked into it, The `outputPath` option is available only in the build schema and not to the make/package schemas.
You cannot relay on the `angular.json` file since these value can be overridden by the user when they invoke the builder.

I think it is best for now to leave it as it is.
If you have an idea how to implement it correctly, I would be more then happy to accept a pull request.

**Comment by bennymeg** [comment]
Fixed in v10.0.0-beta.0
Please use `nx run <electron-app-name>:package --sourcePath='build/apps'`.

Note: since v10 the packager had been replaced with 'electron-packager', if you used static packaging options you will need to use `maker.options.json` instead.
